README.md

```diff
@@ -5,148 +5,51 @@
+The options are documented in detail in [Core Options](https://github.com/embroider-build/embroider/blob/main/packages/core/src/options.ts) and [Compat Options](https://github.com/embroider-build/embroider/blob/main/packages/compat/src/options.ts)
 
 ## Environment variables
 
 For optional features, Embroider supports the following environment variables:
 
-- `EMBROIDER_WORKING_DIRECTORY`: by default Embroider writes internal build-time artifacts like rewritten packages to `node_modules/.embroider`. In the case of running multiple builds concurrently (e.g. building for production and test in parallel) this would cause conflicts when concurrent processes try to write into the same directory. For this case you can point each Embroider process to a different directory using this environment variable. It can be an absolute file path, or relative to the application root directory. 
-
-
-## Configuring asset URLs
-
-If you are serving your assets from a different origin (like a CDN) from where your index.html content will
-be served from, you can use the publicAssetURL option to specify the base URL. In pre-Embroider Ember apps,
-this was accomplished by configuring the `fingerprint: { prepend: ... }` option handled by broccoli-asset-rev.
-
-```js
-return require('@embroider/compat').compatBuild(app, Webpack, {
-  packagerOptions: {
-    publicAssetURL: EmberApp.env() === 'production' ? 'https://your-cdn-here.com/' : '/', // This should be a URL ending in "/"
-  },
-});
-```
-
-## Template Tag Codemod
-
-Edit `ember-cli-build.js`:
-```js
-return require('@embroider/compat').templateTagCodemod(app, {
-  shouldTransformPath: (path) => { return true; },
-  nameHint: (path) => { 
-    // example path: shared/my-button
-    return path; 
-  },
-  dryRun: true,
-});
-```
-Run a normal ember build to transform your hbs templates into template tag single file components.
-Requires optimized build (static* flags to be turned on)
-
-### Options
-
-* `shouldTransformPath` - allows users to filter the templates that the code mod would run on
-* `nameHint` - optional function control the import name and template replacement values - valid JS identifier required or it will be coerced into one
-* `dryRun` - option can be used to obtain a summary of the changed the build would perform and which files it would act upon
-
-### Limitations
-
-* App templates only
-* `@embroider/compat` >= 3.6.0
+- `EMBROIDER_WORKING_DIRECTORY`: by default Embroider writes internal build-time artifacts like rewritten packages to `node_modules/.embroider`. In the case of running multiple builds concurrently (e.g. building for production and test in parallel) this would cause conflicts when concurrent processes try to write into the same directory. For this case you can point each Embroider process to a different directory using this environment variable. It can be an absolute file path, or relative to the application root directory.
 
 ## Compatibility
```

Comment:
GH won't let me comment on the untouched lines but below there is a 'With Classic Builds' header that seems off

Reply:
yea you're right, we actually don't yet have an answer for engines so we can just remove the whole section. Thanks for the catch 🎉